src/openrct2/world/Map.cpp

```diff
@@ -1254,10 +1254,13 @@ void map_obstruction_set_error_text(TileElement* tileElement, GameActionResult&
             }
             break;
         case TILE_ELEMENT_TYPE_SMALL_SCENERY:
+        {
```

Comment:
The changes in this file seems like they belong to another commit

Reply:
Er, no. Creating a variable in a `case` requires { and }.

Reply:
No no, I meant that the changes you made in this file are not using `Formatter::Common()`, so it's a bit misleading with the PR title and commit messages. I'd just cherry pick the changes in here to another commit and rephrase the PR

Reply:
:(